lang/wren/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem        1.0
+PortGroup         github 1.0
+PortGroup         makefile 1.0
+
+github.setup      wren-lang wren 0.3.0
+
+categories        lang
+license           MIT
+platforms         darwin
+
+description       Wren is a small, fast, class-based concurrent scripting \
+                  language.
+
+long_description  {*}${description} Think Smalltalk in a Lua-sized package \
+                  with a dash of Erlang and wrapped up in a familiar, modern \
+                  syntax.  Wren is intended for embedding in applications. It \
+                  has no dependencies, a small standard library, and an \
+                  easy-to-use C API. It compiles cleanly as C99, C++98 or \
+                  anything later.
+
+homepage          https://wren.io
+
+maintainers       {gmail.com:herby.gillot @herbygillot} \
+                  openmaintainer
+
+checksums         rmd160  2b0acc5b21184ff5f8bb485def843418e9a7ca3f \
+                  sha256  489ac252ec805bfa7aab07286e2bc8dc6348bdb55857f7afb9640c5919e118ed \
+                  size    1228925
+
+worksrcdir        ${worksrcdir}/projects/make.mac
+
+build.args-append verbose=1
+build.target      wren_shared
+
+if {${os.arch} eq "i386"} {
+    build.args-append config=release_32bit
+} else {
+    build.args-append config=release_64bit
+}
+
+post-build {
+    system -W ${worksrcpath}/../../lib \
+        "install_name_tool  -id \"${prefix}/lib/libwren.dylib\" libwren.dylib"
```

Comment:
Is it possible to fix this in the build system instead, by using `-install_name (insert prefix here)/lib/libwren.dylib`?

Reply:
Think I was wrong in including this.  Removed.